src/Raven.Studio/typescript/components/common/ButtonGroupWithLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ButtonGroup, ButtonGroupProps, Label } from "reactstrap";
```

Comment:
Are you using Label in this import?

Reply:
Not anymore - I forgot to remove it. It's gone by now.